x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/rules/logical/TranslateTimeSeriesAggregate.java

```diff
@@ -165,15 +165,12 @@ LogicalPlan translate(TimeSeriesAggregate aggregate) {
                 if (attr.name().equals(MetadataAttribute.TSID_FIELD)) {
                     tsid.set(attr);
                 }
-                if (attr.name().equals(MetadataAttribute.TIMESTAMP_FIELD)) {
-                    timestamp.set(attr);
-                }
```

Comment:
I consider it a happy bonus of this PR that we are no longer fishing out the timestamp as part of the translate rule.  I wonder if we should do something similar for the TSID.  It just seems wrong to be doing this here.